doc/module-docs/cc_timezone/example1.yaml

```
@@ -0,0 +1,2 @@
+#cloud-config
+{timezone: US/Eastern}
```

Prefer manual formatting
```suggestion
#cloud-config
timezone: US/Eastern
```